src/main/java/mate/academy/Main.java

```diff
@@ -1,23 +1,27 @@
-package mate.academy;
-
-import mate.academy.exception.AuthenticationException;
-import mate.academy.model.User;
-import mate.academy.service.AuthenticationService;
-import mate.academy.service.AuthenticationServiceImpl;
-import mate.academy.service.OrderService;
-import mate.academy.service.OrderServiceImpl;
-
-public class Main {
-    public static void main(String[] args) {
-        AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
-        try {
-            user = authenticationService.login("bob", "1234");
-        } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
-        }
-        OrderService orderService = new OrderServiceImpl();
-        orderService.completeOrder(user.getUserId());
-    }
-}
+package mate.academy;
+
+import mate.academy.exception.AuthenticationException;
+import mate.academy.model.User;
+import mate.academy.service.AuthenticationService;
+import mate.academy.service.AuthenticationServiceImpl;
+import mate.academy.service.OrderService;
+import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        AuthenticationService authenticationService = new AuthenticationServiceImpl();
+        User user;
+        try {
+            user = authenticationService.login("bob", "1234");
+        } catch (AuthenticationException e) {
+            logger.error("Method login doesn't work successfully", e);

```

Comment:
this is not informative enough. need to specify what happened and why method failed